scripts/ntrip_ros.py

```diff
@@ -102,15 +126,27 @@ def subscribe_nmea(self, nmea):
     self._client.send_nmea(nmea.sentence)
 
   def publish_rtcm(self, event):
-    for raw_rtcm in self._client.recv_rtcm():
-      self._rtcm_pub.publish(RTCM(
-        header=Header(
-          stamp=rospy.Time.now(),
-          frame_id=self._rtcm_frame_id
-        ),
-        data=raw_rtcm
-      ))
-
+    try:
+      for raw_rtcm in self._client.recv_rtcm():
+        if self.rtcm_msg_pkg == 'mavros_msgs':
+          rtcm_msg = RTCM(
+            header=Header(
+              stamp=rospy.Time.now(),
+              frame_id=self._rtcm_frame_id
+            ),
+            data=raw_rtcm
+          )
+        elif self.rtcm_msg_pkg == 'rtcm_msgs':
+          rtcm_msg = Message(
+            header=Header(
+              stamp=rospy.Time.now(),
+              frame_id=self._rtcm_frame_id
+            ),
+            message =raw_rtcm
+          )
+        self._rtcm_pub.publish(rtcm_msg)
+    except:
+      pass
```

Comment:
Would probably be good to log the error here. That can get a little noisy, but failing silently may cause confusion